buildlib/az-distro-release.yml

```diff
@@ -112,3 +112,6 @@ jobs:
           addChangeLog: false
           assetUploadMode: replace
           assets: "./$(artifact_name)"
+      - task: PostBuildCleanup@3
+        displayName: 'Clean Agent Directories'
+        condition: always()
```

Comment:
why needed?

Reply:
for free space after build, may be not so need

Reply:
i think we can remove since it runs in docker?